src/test/java/server/RequestParserTest.java

```diff
@@ -23,9 +23,9 @@
     @Before
     public void stubRequest() {
         requestLine = "GET /redirect HTTP/1.1";
-        requestHeaders = "User-Agent: Ruby\n" + "Connection: close\n" + "Host: 127.0.0.1:5000\n" + "Content-Length: 46";
+        requestHeaders = "User-Agent: Ruby\r\n" + "Connection: close\r\n" + "Host: 127.0.0.1:5000\r\n" + "Content-Length: 46\r\n";
```

Comment:
I missed this before, and doesn't really matter, but why is the user agent Ruby?

Reply:
Because I just copied one of the requests made in the acceptance tests.